components/button/index.js

```diff
@@ -13,11 +13,13 @@ function Button( { href, isPrimary, isLarge, isToggled, className, ...additional
 	} );
 
 	const tag = href !== undefined ? 'a' : 'button';
-	const tagProps = tag === 'a' ? { href } : { type: 'button' };
+	const tagProps = tag === 'a' ? { href, target } : { type: 'button', disabled };
+	const disabledLinkProps = tag === 'a' && !! disabled ? { 'aria-disabled': true, tabIndex: '-1' } : {};
```

Comment:
I don't think there's usage of it yet, but by this button it would appear a disabled link button would still navigate.

I wonder if it would be acceptable to render a disabled link as a button, since buttons can support the `disabled` attribute and `href` is presumably non-relevant on a disabled link anyways ([example](https://github.com/aduth/dones/blob/master/src/components/button/index.js)).